app/src/main/java/ru/practicum/homework/ui/view/ContactDescription.kt

```diff
@@ -0,0 +1,41 @@
+package ru.practicum.homework.ui.view
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import ru.practicum.homework.R
+import ru.practicum.homework.ui.utils.VerticalSpacer
+
+@Composable
+fun ContactDescription(
+    modifier: Modifier = Modifier,
+    phone: String,
+    address: String,
+    email: String?,
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+
+        InfoRow(
+            title = R.string.phone, value = phone
```

Comment:
Если параметров функции больше 1 - лучше делать каждую на новой строке 
```suggestion
            title = R.string.phone, 
            value = phone
```